tests/Cosmonaut.System/CosmosProvisioningTests.cs

```diff
@@ -153,6 +156,50 @@ public async Task CosmosStoreSettings_CreatesCollectionWithThroughput_WhenThroug
             collectionOffer.Content.OfferThroughput.Should().Be(10000);
         }
 
+        [Fact]
+        public async Task CosmosStoreSettings_CreatesCollectionWithUniqueKeyPolicy()
+        {
+            var cosmosStoreSettings = new CosmosStoreSettings(_databaseId, _emulatorUri, _emulatorKey, settings =>
```

Comment:
Indentation looks weird on this test. Can you fix the formatting please? I'm about to merge this.